cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/aggregation/NGSIGenericColumnAggregator.java

```diff
@@ -98,7 +98,7 @@ public void aggregate(NGSIEvent event) {
             JsonElement attrValue = contextAttribute.getValue();
             String attrMetadata = contextAttribute.getContextMetadata();
             LOGGER.debug("[" + getName() + "] Processing context attribute (name=" + attrName + ", type=" + attrType + ")");
-            if (isEnableGeoParse() && (attrType.equals("geo:json") || attrType.equals("geo:point"))) {
+            if (isEnableGeoParse() && (attrType.equalsIgnoreCase("geo:json") || attrType.equalsIgnoreCase("geo:point"))) {
```

Comment:
CNR entry missing. Maybe something like this:

```
- [...] Ignore case for attribute types geo:json and geo:point interpretation
```

Reply:
Fixed in [cfef8e8](https://github.com/telefonicaid/fiware-cygnus/pull/1877/commits/cfef8e80cfa1b5324073bffd762e43afce58d21f)